### PR TITLE
Transformation updates

### PR DESCRIPTION
**Pull request by OlliPaj**
As requested

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,12 +36,12 @@
...
 	},
 	"dependencies": {
-		"@natlibfi/marc-record": "^3.0.0",
+		"@natlibfi/marc-record": "^4.0.0",
 		"@natlibfi/marc-record-validate": "^4.0.0",
-		"@natlibfi/marc-record-validators-melinda": "^5.1.1",
-		"@natlibfi/melinda-record-import-commons": "^1.5.0",
+		"@natlibfi/marc-record-validators-melinda": "^6.0.0",
+		"@natlibfi/melinda-commons": "^4.0.4",
+		"@natlibfi/melinda-record-import-commons": "^3.1.1",
```
Comment (natlibfi-arlehiko) [inline package.json]: Update to the newest version (API authentication has changed and the new version of API client handles this).

Comment (OlliPaj) [reply package.json]: Updated.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  image: plugins/docker
+  settings:
+    repo: quay.io/natlibfi/melinda-record-import-transformer-publication-archives
+    registry: quay.io
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+  when:
+    branch: [master, test]
```
Comment (natlibfi-arlehiko) [inline .drone.yml]: Change this to `[dev]` until we are in production.

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
   - 'node'
-  - '7'
+  - '8'
```
Comment (natlibfi-arlehiko) [inline .travis.yml]: Change to `10` which is the current LTS version.

File: package.json [hunk 4]
```diff
@@ -17,51 +17,50 @@
 	"version": "1.0.0",
 	"main": "./dist/index.js",
 	"engines": {
-		"node": ">=7"
+		"node": ">=8"
```
Comment (natlibfi-arlehiko) [inline package.json]: Change to `10` which is the current LTS version.

File: package.json [hunk 5]
```diff
@@ -17,51 +17,50 @@
...
 	"dependencies": {
-		"@natlibfi/marc-record": "^3.0.0",
-		"@natlibfi/marc-record-validate": "^4.0.0",
-		"@natlibfi/marc-record-validators-melinda": "^5.1.1",
-		"@natlibfi/melinda-record-import-commons": "^1.5.0",
+		"@natlibfi/marc-record": "^4.0.0",
+		"@natlibfi/marc-record-validate": "^5.0.0",
+		"@natlibfi/marc-record-validators-melinda": "^7.0.4",
+		"@natlibfi/melinda-commons": "^7.0.1",
+		"@natlibfi/melinda-record-import-commons": "6.0.2",
```
Comment (natlibfi-arlehiko) [inline package.json]: Use semver caret range: `^6.0.0`

File: package.json [hunk 6]
```diff
@@ -17,51 +17,50 @@
...
 		"chai-as-promised": "^7.1.1",
-		"cross-env": "^5.2.0",
-		"eslint": "^5.9.0",
-		"eslint-config-xo": "^0.25.0",
-		"eslint-plugin-import": "^2.14.0",
-		"mocha": "^5.2.0",
-		"nodemon": "^1.18.6",
+		"cross-env": "^6.0.0",
+		"eslint": "^6.4.0",
+		"eslint-config-xo": "^0.26.0",
```
Comment (natlibfi-arlehiko) [inline package.json]: Replace with the following: `>=0.26.0 <1.0.0` (Otherwise minor version updates are not applied)

File: package.json [hunk 7]
```diff
@@ -76,15 +75,18 @@
...
 	},
 	"babel": {
 		"presets": [
 			[
-				"env",
+				"@babel/preset-env",
 				{
 					"targets": {
-						"node": "7"
+						"node": "8"
```
Comment (natlibfi-arlehiko) [inline package.json]: Change to `10` which is the current LTS version.

**Pull request merged**